projects/1.21/assets/flapjacks-economy/jackseconomy/lang/zh_cn.json

```
@@ -0,0 +1,150 @@
+  "curios.identifier.wallet": "钱包",
+  "jackseconomy.value": "面额: %s",
+  "jackseconomy.wallet_balance": "钱包: %s",
+  "key.jackseconomy.open_wallet": "打开钱包",
+  "key.categories.jackseconomy": "Jack的经济",
+  "jackseconomy.balance_out_of": "余额: %s/%s",
+  "jackseconomy.fe_per_tick": "功耗: %s FE/tick",
+  "jackseconomy.progress_per_tick": "速度: %s 单位进度/tick",
+  "jackseconomy.subtitle.coin": "硬币：叮当",
+  "jackseconomy.subtitle.cash": "纸币：翻动",
+  "jackseconomy.subtitle.checkout": "管理员商店：结账",
+  "block.jackseconomy.currency_converter": "货币转换者",
+  "jackseconomy.items": "物品: %s",
+  "jackseconomy.in_cart": "在购物车中: %s",
+  "jackseconomy.to_sell": "待售: %s",
+  "jackseconomy.no_wallet": "没有钱包！",
+  "jackseconomy.purchase": "购入",
+  "jackseconomy.cant_afford": "买不起！",
+  "jackseconomy.shopping_cart_empty": "购物车是空的！",
+  "jackseconomy.sold_items": "已售出物品: %s",
+  "jackseconomy.bought_items": "已购买物品: %s",
+  "jackseconomy.total": "总计: %s",
+  "jackseconomy.hold_an_item": "你需要手持一个物品！",
+  "jackseconomy.importer_price_set": "%s的进口者买价已设置为%s！",
+  "jackseconomy.exporter_price_set": "%s的出口者售价已设置为%s！",
+  "jackseconomy.importer_price_removed": "%s的进口者买价已移除！",
+  "jackseconomy.exporter_price_removed": "%s的出口者售价已移除！",
+  "jackseconomy.generate_this_ticket": "您可以使用以下命令生成此清单：",
+  "jackseconomy.copy": "[复制]",
+  "jackseconomy.click_to_copy": "点击这里复制命令！",
+  "jackseconomy.config.hide_price_tooltips": "隐藏价格提示: %s",
+  "jackseconomy.config.alternative_tooltip_format": "替代的工具提示格式: %s",
+  "jackseconomy.config.wallet_hud_position_right": "钱包HUD在右侧的位置: %s",
+  "jackseconomy.config.wallet_hud_position_y_offset": "钱包HUD的Y轴偏移量: %s",
+  "jackseconomy.true": "开启",
+  "jackseconomy.false": "关闭",
+  "jackseconomy.golden_exporter_manifest_description": "允许在出口者中出售任何可出售的物品",
+  "jackseconomy.edit": "编辑",
+  "jackseconomy.right_click_to_edit_price": "右键点击编辑价格",
+  "jackseconomy.middle_click_to_remove": "中键点击移除物品",
+  "jackseconomy.no_price": "没有设置价格",
+  "jackseconomy.right_click_to_rename": "右键点击重命名",
+  "jackseconomy.middle_click_to_remove_category": "中键点击以移除分类",
+  "jackseconomy.click_to_open_admin_shop": "点击打开管理员商店！",
+  "jackseconomy.shift_right_click_to_rename": "L-Shift键 + 右键点击重命名",
+  "jackseconomy.manifest_required": "交易需要清单",
+  "jackseconomy.leave": "离开",
+  "jackseconomy.stay": "保持",
+  "jackseconomy.admin_shop_exit_prompt": "你确定要退出商店吗？你当前的购物车将会丢失。",
+  "jackseconomy.configuration": "配置",
+  "jackseconomy.direction.north": "北 (正面)",
+  "jackseconomy.direction.south": "南 (背面)",
+  "jackseconomy.direction.east": "东 (左面)",
+  "jackseconomy.direction.west": "西 (右面)",
+  "jackseconomy.direction.up": "顶面",
+  "jackseconomy.direction.down": "底面",
+  "jackseconomy.sidevalue.input": "输入",
+  "jackseconomy.sidevalue.output": "输出",
+  "jackseconomy.sidevalue.rejection_output": "拒绝输出",
+  "jackseconomy.sidevalue.none": "无",
+  "block.jackseconomy.mechanical_exporter": "动力进口者",
+  "block.jackseconomy.mechanical_importer": "动力出口者",
+  "jackseconomy.max_storage": "大存储量: %s（已使用: %s）",
```

```suggestion
  "jackseconomy.max_storage": "大存储量：%s（已使用：%s）",
```
【要求】标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。